src/lib/isContainerID.js

```diff
@@ -0,0 +1,28 @@
+import assertString from './util/assertString';
+
+// https://en.wikipedia.org/wiki/ISO_6346
+const isContainerIdReg = new RegExp('^[A-Z]{3}U[0-9]{7}$');
+const isDigit = new RegExp('^[0-9]{1}');
+
+export default function isContainerID(str) {
+  assertString(str);
+  str = str.trim();
```

Comment:
I have the same concerns about this, as I have mentioned in your other PR:
https://github.com/validatorjs/validator.js/pull/2143

what do you think?

Reply:
You are right @pano9000 -- I don't think we should be doing cleaning behind the validators, let the users do their own cleaning. @songyuew pls update.